src/pages/VenuePage/VenuePage.tsx

```diff
@@ -295,7 +295,7 @@ const VenuePage = () => {
     return <LoadingPage />;
   }
 
-  if (profile && !(profile?.partyName && profile?.pictureUrl)) {
+  if (profile && !isCompleteProfile(profile)) {
```

Comment:
`isCompleteProfile` is an attempt to make this code a little more self documenting.

Reply:
Excellent! This is what I wanted in my retroactive review of https://github.com/sparkletown/sparkle/pull/932#pullrequestreview-547762211